### PR TITLE
fix(HentaiZ): change domain

### LINKED ISSUE #4879: HentaiZ | hentaiz.*
**Describe the Bug**
The Hentaiz presense not working anymore because hentaiz has change the domain from hentaiz.net to hentaiz.vip ( hentaiz.to)
**Steps to Reproduce**

1. https://hentaiz.vip/
2. change domain in presense from hentaiz.net to hentaiz.vip

**Expected Behavior**
The presence for hentaiz is working angain

**Screenshots**
image

**Environment**
- OS:  Windows 10, Windows 8.1, Windows 7 or Linux 
- Browser: Edge
- Version: 95.0.1020.44 (64-bit)

**Additional context**
nothing

### PR DESCRIPTION
**Pull request by kyrie25**
**Describe the changes in this pull request:**
Resolves #4879 

~~I don't know why and don't want to know why but 450 people are using this presence so~~

<details>
<summary> Proof of the modification working: </summary>
<br>

image
image

</details>

### CODE REVIEW

**Review by Slowlife01 — DISMISSED** [review]

**Comment by kyrie25** [comment]
I misunderstood the presence owner's intention

- Fixed the presence so that it'd detect pages:
image
- Changes how the movies are displayed:
Discord_bViRlzC5nC

File: websites/H/HentaiZ/presence.ts [hunk 1]
```diff
@@ -37,7 +36,7 @@ presence.on("UpdateData", async () => {
     browsingTimestamp = Math.floor(Date.now() / 1000);
   }
 
-  if (!playback) {
+  if (!document.getElementsByClassName("mr-1")[0].querySelector("h2")) {
```
Comment (Slowlife01) [inline websites/H/HentaiZ/presence.ts:39]: Is this basically just

```suggestion
  if (!document.querySelector(".mr-1 h2")) {
```

Comment (kyrie25) [reply websites/H/HentaiZ/presence.ts:39]: Will test ASAP

Comment (kyrie25) [reply websites/H/HentaiZ/presence.ts:39]: Yes it is

Comment (kyrie25) [reply websites/H/HentaiZ/presence.ts]: Yes it is, will fix ASAP

File: websites/H/HentaiZ/presence.ts [hunk 2]
```diff
@@ -24,8 +24,7 @@ presence.on(
 
 presence.on("UpdateData", async () => {
   const curPath = document.location.pathname,
-    playback =
-      video !== null && !isNaN(video.duration) && curPath.includes("/xem-phim"),
+    playback = video !== null && !isNaN(video.duration),
```
Comment (EncryptedDev) [inline websites/H/HentaiZ/presence.ts:27]:
```suggestion
    playback = !isNaN(video?.duration),
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**